rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml

```diff
@@ -0,0 +1,30 @@
+title: Lolbin Unregmp2.exe Use As Proxy
+id: 727454c0-d851-48b0-8b89-385611ab0704
+status: experimental
+description: Detect use of unregmp2.exe as proxy to launch a custom wmpnscfg.exe
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Unregmp2/
+author: frack113
+date: 2022/12/29
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.execution
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+       - Image|endswith: '\unregmp2.exe'
+       - OriginalFileName: 'unregmp2.exe'
+    selection_cmd:
+        CommandLine|contains: ' /HideWMP'
+    condition: all of selection_*
+fields:
+    - CommandLine
+    - CurrentDirectory 
+    - ParentCommandLine
+    - User
```

Comment:
As we've seen in one issue this `fields` field can be confusing. I suggest we don't use it for now.
```suggestion
```